src/auspex/instruments/yokogawa.py

```diff
@@ -21,19 +22,20 @@ class YokogawaGS200(SCPIInstrument):
     output_range       = FloatCommand(scpi_string=":source:range")
     protection_volts   = FloatCommand(scpi_string=":source:protection:voltage")
     protection_current = FloatCommand(scpi_string=":source:protection:current")
-    sense              = BoolCommand(scpi_string=":sense:state", value_map={True: "1", False: "0"})
-    output             = BoolCommand(scpi_string=":output:state", value_map={True: "1", False: "0"})
+    sense              = StringCommand(scpi_string=":sense:state", value_map={True: "1", False: "0"})
+    output             = StringCommand(scpi_string=":output:state", value_map={True: "1", False: "0"})
     sense_value        = FloatCommand(get_string=":fetch?")
     averaging_nplc     = IntCommand(scpi_string=":sense:nplc") # Number of power level cycles (60Hz)
-    ramp               = RampCommand(increment=1e-4, pause=20e-3, scpi_string=":source:level", value_range=(-100e-3,100e-3))
+ #   ramp               = RampCommand(increment=1e-4, pause=20e-3, scpi_string=":source:level", value_range=(-100e-3,100e-3))
```

Comment:
Why get rid of this one?

Reply:
This PR had a lot more than it says :) The only intended commit was the one on standalone instruments, but to be verified by @Himorask .